sdcm/tester.py

```diff
@@ -280,7 +280,10 @@ class ClusterTester(db_stats.TestStatsMixin, unittest.TestCase):  # pylint: disa
     monitors: BaseMonitorSet = None
     loaders: Union[BaseLoaderSet, LoaderSetAWS, LoaderSetGCE] = None
     db_cluster: Union[BaseCluster, BaseScyllaCluster] = None
-    k8s_cluster: Union[eks.EksCluster, gke.GkeCluster, mini_k8s.LocalKindCluster, None] = None
+
+    @property
+    def k8s_cluster(self):
```

Comment:
Isnt this gonna hide places we didn't handle multi clusters correctly ?

Reply:
We can remove it and update the `upgrade_test` module.
This is the only place where it is used.
Without it the PR would be bloated much more.